python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -147,13 +147,26 @@ def callback(self, pre, post, node_map):
         act = node_map[self.act][0]
         conv_pool = node_map[self.pattern][0]
         pad = node_map[self.pad][0]
+
         if not all(conv_pool.attrs.padding[i] == 0 for i in range(len(conv_pool.attrs.padding))) \
            or not isinstance(pad.args[1], tvm.relay.Constant) or not pad.args[1].data.shape == () \
            or not int(pad.args[1].data.numpy()) == 0:
            return
 
         pad_width = pad.attrs.pad_width
-        padding = list(pad_width[-2]) + list(pad_width[-3]) # left, right, top, bottom
+        top = pad_width[-2][0]
+        bottom = pad_width[-2][1]
+        left = pad_width[-1][0]
+        right = pad_width[-1][1]
+
+        # Different padding values currently not supported in TTNN
+        # For Conv2dOp Op, refer third_party/tt-mlir/lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp:578
+        # For Maxpool2d Op, refer third_party/tt-mlir/lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp:614
+        assert top == bottom, "padding top and padding bottom must be same"
```

Comment:
Let's assert with the following message: "Unsupported different top and bottom padding size". Let's set a similar msg to the second assert as well